internal/learn/the-community/how-to-support.md

```diff
@@ -9,21 +9,49 @@ As ecosystem members there are different ways to support the vision.
 
 This page is a non-exhaustive collection of ideas and smaller to bigger ways to support the growth and expansion of the ecosystem.
 
-## Basic level
-
-These activities require to register accounts on different platforms and time.
-
 ## Join the IOTA Discord
 
 **First and foremost** join the IOTA Community on [the IOTA Discord](discord.md) server. Click on [this link](http://discord.iota.org/) to get the invitation to the server and get in touch with the IOTA Foundation members and the awesome IOTA community members.
 
+## Crypto platforms
+
+### Coinmarketcap
+
+#### Watchlist
+
+Register an account or use your existing Binance.com account.
+Add [IOTA](https://coinmarketcap.com/currencies/iota/) and [Shimmer](https://coinmarketcap.com/currencies/shimmer/) to your watchlist by clicking on the ⭐
+
+![coinmarketcap_iota_watchlist.png](/img/participate/how-to-support/coinmarketcap_iota_watchlist.png)
```

Comment:
I think for SEO it is better to use a descriptive text. Also for accessibility. Maybe you want to change that 🤔